### ISSUE TITLE
aria-live region gives incorrect instructions in one use case

### ISSUE THREAD

**Issue by jkjustjoshing** [JedWatson/react-select#3685]
Using VoiceOver in Safari on macOS 10.14.5

In the "Popout" example at https://react-select.com/advanced#experimental, VoiceOver announces "Press Tab to select the option and exit the menu.". However, pressing tab neither selects the current option, nor exits the menu. 

There should be a way to pass in a template to replace the aria-live region template, for when I want to change the instructions given to my screen reader users.

This would also address #3352

**Comment by ebonow** [comment]
I believe this is a duplicate of #3352 and additionally will be resolved by https://github.com/JedWatson/react-select/pull/4161

Closing this as a duplicate and addressing with the cited issue and PR.
